src/main/java/teammates/common/datatransfer/FeedbackQuestionDetails.java

```diff
@@ -4,6 +4,8 @@
 import java.util.List;
 import java.util.Map;
 
+import teammates.common.datatransfer.attributes.FeedbackQuestionAttributes;
+import teammates.common.datatransfer.attributes.FeedbackResponseAttributes;
```

Comment:
This is the base class of `Feedback*QuestionDetails`; should be under `.questions`?